frontend/projects/ui/src/app/components/toast-container/refresh-alert/refresh-alert.component.html

```diff
@@ -1,9 +1,32 @@
-<alert *ngIf="show$ | async" header="Refresh Needed" (dismiss)="onDismiss()">
-  Your user interface is cached and out of date. Hard refresh the page to get
-  the latest UI.
-  <ul>
-    <li><b>On Mac</b>: cmd + shift + R</li>
-    <li><b>On Linux/Windows</b>: ctrl + shift + R</li>
-  </ul>
-  <a alertButton class="enter-click" role="cancel">Ok</a>
-</alert>
+<div *ngIf="!onPwa; else pwa">
+  <alert *ngIf="show$ | async" header="Refresh Needed" (dismiss)="onDismiss()">
```

Comment:
Could only do `show$ | async` once if you elevate in scope

Reply:
I actually cannot due to the way ionic styling is applied to the buttons and decided it was not worth it to reimplement myself.